web/src/lib/components/shared-components/change-date.svelte

```diff
@@ -57,6 +57,8 @@
   const handleKeydown = (event: KeyboardEvent) => {
     if (['ArrowUp', 'ArrowDown', 'ArrowLeft', 'ArrowRight'].includes(event.key)) {
       event.stopPropagation();
+    } else {
+      event.stopPropagation();
     }
```

Comment:
This right now could be reduced to `=> event.stopPropagation()`, right? :P
I don't think that's what is wanted here...?

Reply:
@danieldietzler I'll reduce that code to one line. I think the delete key issue is caused by event bubbling, is there anything I need to add?

Reply:
I don't think we want to do event propagation for every single event.